tls: use the most recently added matching SecureContext in default SN… (#36072)
nodejs/node: doc/api/tls.md

The `server.addContext()` method adds a secure context that will be used if
the client request's SNI name matches the supplied `hostname` (or wildcard).

> When there are multiple matching contexts, the most recently added one will be

Reviewer: ```suggestion
When there are multiple matching contexts, the most recently added one is
```
